conf/gunicorn/websocket.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+
+bind = "unix:/tmp/gunicorn-websocket.sock"
+worker_class = "h.websocket.Worker"
+graceful_timeout = 0
+worker_connections = 4096
+proc_name = "websocket"
+
+
+if "GUNICORN_TIMEOUT" in os.environ:
+    timeout = int(os.environ["GUNICORN_TIMEOUT"])
```

Comment:
This duplicates the `GUNICORN_TIMEOUT` code from `app.py` but it's two lines, I don't think it's worth introducing a shared file for